draft-ietf-taps-interface.md

```diff
@@ -2151,6 +2151,32 @@ does not provide message boundary preservation, and since many of these
 protocols require message boundaries to function, each application layer
 protocol has defined its own framing.
 
+Message Framers are loaded into Connections when specified in a Preconnection.
+When Message Framers are being used, they can interact with the setup and
+teardown of a Connection. For example, a Framer can start sending data
```

Comment:
To me this sounds a bit like Framers are only coming into effect when/after the Preconnection becomes a Connection. Also, this makes me wonder what "load into Connections" means.

Maybe let's rather say something like:
"To use a Message Framer, an application has to add it to a Preconnection.
Then, the Message Framer can interact with the setup and teardown of the Connection. ..."

Reply:
Reworked this text, let me know if that works!

Reply:
Better, I think